apps/src/applab/designMode.js

```diff
@@ -113,6 +113,9 @@ designMode.resetElementTray = function (allowEditing) {
  */
 designMode.onPropertyChange = function(element, name, value) {
   var handled = true;
+  var to_px = function (inp) {
+    return inp ? inp + 'px' : '';
+  };
```

Comment:
I might declare this outside of onPropertyChange, just bc onPropertyChange can be called a lot and each call will be creating a new function to_px right now. Also our js style uses camelCase instead of snake_case so toPx instead of to_px (or maybe something like appendPx?)